inscriptions/ISic020539.xml

```diff
@@ -0,0 +1,218 @@
+                                <dimensions type="letterHeight">
+                                    <height unit="mm"/>
+                                </dimensions>
+                            </handNote>
+                        </handDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origPlace>
+                                <placeName type="ancient" ref="https://pleiades.stoa.org/places/462214">Gela</placeName>
+                                <placeName type="modern">Gela</placeName>
+                                <geo>37.0630925, 14.2584705</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" notBefore-custom="-0450" notAfter-custom="-0400">450-400 BC</origDate>
+                        </origin>
+                        <provenance type="found">Found in Gela<geo>37.0630925, 14.2584705</geo>.</provenance>
+                        <provenance type="observed" subtype="autopsied">No Autopsy</provenance>
+                        <acquisition>Gela, Museo Archeologico Regionale di Gela</acquisition>
+                    </history>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Encoded following EpiDoc guidelines 9.2</p>
+            <xi:include href="../alists/ISicily-taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for ISicily controlled values</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <calendarDesc>
+                <calendar xml:id="julian">
+                    <p>Julian Calendar</p>
+                </calendar>
+            </calendarDesc>
+            <langUsage>
+                <language ident="en">English</language> 
+                <language ident="it">Italian</language> 
+                <language ident="grc">Ancient Greek</language> 
+                <language ident="la">Latin</language> 
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.ownership" cert="low" ref="https://www.eagle-network.eu/voc/typeins/lod/104.html">ownership</term> <!-- function for literary quotations/mentions of mythological figures? we can add another function as https://www.eagle-network.eu/voc/typeins/lod/55.html (if it is a function and if this interpratation is correct...)? -->
```

Comment:
No, this would be 'label', which in EAGLE i think is only: https://www.eagle-network.eu/voc/typeins/lod/109.html (note the erroneous 'same as' for French signature d'artiste', which is patently isn't... sigh...). it probably is 'same as' https://www.eagle-network.eu/voc/typeins/lod/427.html, but they don't cross reference each other... sigh... otherwise, this is indeed a case for https://www.eagle-network.eu/voc/typeins/lod/105.html

Reply:
I changed as https://www.eagle-network.eu/voc/typeins/lod/105.html but I keep the comment open to discuss this category if you like